backend/src/nodes/nodes/image_filter/blur_bilateral.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+import cv2
+import numpy as np
+
+from . import category as ImageFilterCategory
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import ImageInput, NumberInput
+from ...properties.outputs import ImageOutput
+from ...utils.utils import get_h_w_c
+
+
+@NodeFactory.register("chainner:image:bilateral_blur")
+class BlurNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Apply surface/bilateral blur to an image."
+        self.inputs = [
+            ImageInput(),
+            NumberInput("Radius", controls_step=1),
+            NumberInput("Color Sigma", controls_step=1, default=75, maximum=100),
+            NumberInput("Space Sigma", controls_step=1, default=75, maximum=100),
+        ]
+        self.outputs = [ImageOutput(image_type="Input0")]
+        self.category = ImageFilterCategory
+        self.name = "Surface Blur"
+        self.icon = "MdBlurOn"
+        self.sub = "Blur"
+
+    def run(
+        self,
+        img: np.ndarray,
+        radius: int,
+        sigma_color: int,
+        sigma_space: int,
+    ) -> np.ndarray:
+
+        if radius == 0:
+            return img
+
+        img = (img * 255.0).astype(np.uint8)
```

Comment:
Okay, after some research and testing, I believe this is unnecessary. To get it to work with 0-1, you have to divide *only* sigmaColor by 255 to normalize it (but not sigmaSpace, it would seem). Unless I'm tired and messed up my testing, this should give equivalent results without destroying data in the image.